### ISSUE TITLE
valid-describe: warn if first argument is a function

### ISSUE THREAD

**Issue by macklinu** [jest-community/eslint-plugin-jest#61]
From https://github.com/jest-community/eslint-plugin-jest/pull/57#discussion_r167249781

Report in the `valid-describe` rule for the following case:

```js
describe(() => {})
```

**Comment by SimenB** [comment]
Also see https://github.com/facebook/jest/issues/5493

**Comment by macklinu** [comment]
I'll open a PR for this.

**Comment by brianlmacdonald** [comment]
Can I call dibs on this?

**Comment by macklinu** [comment]
@brianlmacdonald go for it - thanks!

I noticed that the `valid-describe` rule falsely reports this as valid:

https://github.com/jest-community/eslint-plugin-jest/blob/5e01e6b2caa7205ab4f5537ab33ce81feff4dad7/rules/__tests__/valid-describe.test.js#L14

when I believe that `describe()` always requires two parameters (`name: string, fn: Function`), which is something I missed in #57.

If you're feeling up for correcting that test case in your PR, that would be awesome too. 😄

**Comment by brianlmacdonald** [comment]
Hey @macklinu, I added the appropriate warnings and tests-- the messages are as follows [message: 'First argument must be name', 'message: 'Describe requires name and callback arguments'']. Happy to change them to any house style verabe if you'd rather before I put in the PR. 

Also, I'm not familiar with husky or commitmsg-- can you tell me what these warnings are referring too so I can fix them?
✖   message may not be empty [subject-empty]
✖   type may not be empty [type-empty]

Hope you're having a good Saturday!

**Comment by SimenB** [comment]
The commit message must follow the form of "feat: some added feature". We use these semantic commits to publish from CI.

See http://karma-runner.github.io/2.0/dev/git-commit-msg.html

**Comment by brianlmacdonald** [comment]
Ok, thanks!

**Comment by macklinu** [comment]
Resolved in #65. Closing.
